assembl/views/api/post.py

```diff
@@ -233,29 +228,20 @@ def get_posts(request):
         orphans = Idea._get_orphan_posts_statement(
             discussion_id, True, include_deleted=deleted).subquery("orphans")
         posts = posts.join(orphans, PostClass.id == orphans.c.post_id)
-        ideaContentLinkQuery = ideaContentLinkQuery.join(
-            orphans, PostClass.id == orphans.c.post_id)
```

Comment:
Simplification while I was here: Instead of keeping two queries synchronized, which is error prone, I only create one and repurpose it using
ideaContentLinkQuery = posts.with_entities(...